spec/requests/api/v1/users_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe 'Api::V1::Users', type: :request do
+  let(:role) { FactoryBot.create(:role) }
```

Comment:
Si el factory user ya existe, se puede hacer directamente:
```suggestion
  let(:role) { create(:role) }
```

Reply:
Podes considerar las `build strategies` dependiendo que necesites del factory:
* create: si necesitas evaluar algo, despues del almacenarlo en la base de datos(en el caso de un update o destroy).
* build: si queres crear la instancia pero no necesitas guardarla para validar.
* attributes_for: si solo necesitas un hash con los atributos y no la instancia.